server/osc_dash.py

```
@@ -55,8 +76,157 @@ def api():
+    gas: float = 0.05,
+    limit_windows: int = 0,
+):
+    """Run backtest on selected tick file or all files in run/ticks/."""
+    from backtest import BacktestParams, iter_ticks, replay
+    from backtest.engine import _simulate_window, group_by_cid
+
+    exit_thresh = {
+        "default_5m": exit_default_5m,
+        "default_15m": exit_default_15m,
+    }
+    for slug in ("btc-up-or-down-5m", "btc-up-or-down-15m",
```

Ruff (0.16.2)

[warning] Loop control variable `slug` not used within loop body

Rename unused `slug` to `_slug`

(B007)